.github/workflows/jdk-19.yml

```diff
@@ -0,0 +1,28 @@
+name: jdk-18
```

Comment:
Minor typo

Reply:
Thanks for spotting the typo, I've just pushed a fix.